samples/subsys/usb/testusb/README.rst

```diff
@@ -0,0 +1,54 @@
+Testusb
```

Comment:
Our convention is to add a label at the top of the doc to enable inter-document linking using a :ref:`name`.  So add this to the top:

    .. _testusb_sample::

    Testusb
    #######

Will this sample work in a non Posix environment (on a supported board with a USB port)?  If so, it would be worth mentioning this in the description.